addons/sms/models/sms_sms.py

```diff
@@ -76,6 +76,11 @@ class SmsSms(models.Model):
         'UUID must be unique',
     )
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        self.env.ref('sms.ir_cron_sms_scheduler_action')._trigger()
```

Comment:
Shouldn't we do something when retrying ? Those flows should be updated as well.

Reply:
Creating an SMS will trigger the queue, if the message was already processed somewhere else, it will just be a noop.